### PR TITLE
[DOCFIX] Improve Chinese document docs/cn/Key-Value-System-API.md to make it more native #353

### PR DESCRIPTION
**Pull request by xqx1568**

### CODE REVIEW

**Comment by xqx1568** [comment]
Jenkins, test this please to raise another test.

File: docs/cn/Key-Value-System-API.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Alluxio除了提供[Filesystem API](File-System-API.html) 让应用程序来读
 
 ## 键值存储库配置参数
 
-Alluxio默认配置是禁用键值存储库的，可以通过配置`alluxio.keyvalue.enabled`为true来启用 (see
-[configuration parameters](Configuration-Settings.html))
+Alluxio 默认配置是禁用键值存储库的，可以通过配置`alluxio.keyvalue.enabled`为true来启用 （参考
```
Comment (LuQQiu) [inline docs/cn/Key-Value-System-API.md]: Could you remove the extra space here?
and use `git add -u`, `git commit -m "remove space"` and `git push origin smallfix` to submit the changes?

**Comment by xqx1568** [comment]
Jenkins, test this please to raise another test.

**Comment by xqx1568** [comment]
Jenkins, test this please

**Comment by xqx1568** [comment]
Get you mean.Please check.Thanks.

**Comment by xqx1568** [comment]
I see what you mean and have done.Please check.Thank you.

**Comment by LuQQiu** [comment]
@xqx1568 Thanks for your effort! Please fix the merge conflicts.
You can use `git pull upstream master` in your branch to merge the latest changes in the master branch.
Use `git add -u` `git commit -m "before solve merge conflicts"` to make a commit before solving merge conflicts.
You can then solve the merge conflicts in files, and use `git add -u`, `git commit -m "solved merge conflicts", and "git push origin smallfix" to push the commits.

Thanks!

**Comment by xqx1568** [comment]
Thank for your help!

**Comment by xqx1568** [comment]
Any other problem?

**Pull request merged**